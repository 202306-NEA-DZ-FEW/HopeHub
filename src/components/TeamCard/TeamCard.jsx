import Image from "next/image";
import React from "react";

const TeamCard = ({ image, name, position }) => {
    return (
        <div className='card w-48  shadow-xl mb-4 m-4 bg-Accent hover:bg-Primary hover:shadow-md'>
            <figure>
                <Image
                    src={image}
                    alt={name}
                    width='96'
                    height='42'
                    layout='responsive'
                />
            </figure>
            <div className='card-body text-NeutralBlack'>
                <h2 className='card-title text-l'>{name}</h2>
                <p>{position}</p>
            </div>
        </div>
    );
};

export default TeamCard;
