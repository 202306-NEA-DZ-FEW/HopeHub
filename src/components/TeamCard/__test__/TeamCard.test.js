import renderer from "react-test-renderer";

import TeamCard from "../TeamCard";

it("renders correctly", () => {
    const tree = renderer.create(<TeamCard />).toJSON();
    expect(tree).toMatchSnapshot();
});
