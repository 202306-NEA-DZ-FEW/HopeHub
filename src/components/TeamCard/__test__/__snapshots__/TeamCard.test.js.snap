// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card w-48  shadow-xl mb-4 m-4 bg-Accent hover:bg-Primary hover:shadow-md"
>
  <figure>
    <img
      data-nimg="1"
      decoding="async"
      height={42}
      onError={[Function]}
      onLoad={[Function]}
      src=""
      style={
        Object {
          "color": "transparent",
          "height": "auto",
          "width": "100%",
        }
      }
      width={96}
    />
  </figure>
  <div
    className="card-body text-NeutralBlack"
  >
    <h2
      className="card-title text-l"
    />
    <p />
  </div>
</div>
`;
